src/platform/packages/shared/kbn-visualization-ui-components/components/query_input/filter_query_input.tsx

```diff
@@ -18,11 +18,14 @@ import {
   EuiFlexGroup,
   EuiIconTip,
   EuiPopoverProps,
+  euiTextBreakWord,
+  useEuiFontSize,
 } from '@elastic/eui';
 import type { DataViewBase, Query } from '@kbn/es-query';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
```

Comment:
It seems like the EUI team is asking people to move away from using `euiThemeVars` and instead using the `useEuiTheme` hook. See https://github.com/elastic/kibana/issues/199715 and search for `euiThemeVars`.

Before:

```tsx
import { EuiIcon, EuiCode, EuiText, useEuiTheme } from '@elastic/eui';
height: ${euiThemeVars.euiSizeXS} / 2;
```

After:

```tsx
import { useEuiTheme } from '@elastic/eui'
const { euiTheme } = useEuiTheme();
height: ${euiTheme.size.xs} / 2;
```